packages/ui-radio-input/src/RadioInputGroup/props.ts

```diff
@@ -87,20 +92,27 @@ type RadioInputGroupOwnProps = {
    * any children (ones that aren't `RadioInput` are passed through)
    */
   children?: React.ReactNode
+  /*
```

Comment:
I think one more asterisk might be needed here e.g: /** because I cannot see this description on the preview page?